pages/k8s/1.19/release-notes.md

```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
 For a full list of the changes introduced in Kubernetes 1.19, please see the
 [upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
 
+- MetalLB Operator
+
+MetalLB offers a software network load balancing implementation that allows for 
+LoadBalancing services in Kubernetes. This charm bundle has been made available
+in the charm store to be deployed along Charmed Kubernetes or MicroK8s. This operator
+deploys upstream MetalLB in layer 2 mode. The BGP mode of upstream MetalLB is not
+supported yet. For more information about deploying and operating MetalLB, please
+see the operator [README](https://github.com/charmed-kubernetes/metallb-operator) and the [upstream documentation](https://metallb.universe.tf/).
```

Comment:
We should link to https://ubuntu.com/kubernetes/docs/metallb, and let *that* page include links to the readme and upstream docs.